longevity_ics_test.py

```diff
@@ -0,0 +1,40 @@
+from longevity_test import LongevityTest
+
+DICT_KEYSPACES_COMPRESSION = {'keyspace_lz4': 'LZ4', 'keyspace_deflate': 'Deflate', 'keyspace_snappy': 'Snappy'}
```

Comment:
My personal preference for names like `KEYSPACES_COMPRESSION_MAP` or `KEYSPACES_COMPRESSION_MAPPING`
any I would argue if the name holds the type, it should be at the end of the name.

Reply:
done